src/getValueFunctions.php

```diff
@@ -51,25 +55,29 @@ function getValue($doc, $fieldName, $default = null) {
 
         // Try using getters
         $fieldNameCamel = util\toCamelCase($fieldName);
-        $getters = array(
-            $fieldName, 'get'.$fieldNameCamel, 'is'.$fieldNameCamel, 'has'.$fieldNameCamel, $fieldNameCamel
-        );
+        $getters = [
+            $fieldName,
+            'get' . $fieldNameCamel,
+            'is' . $fieldNameCamel,
+            'has' . $fieldNameCamel,
+            $fieldNameCamel
+        ];
         foreach ($getters as $methodName) {
             if (method_exists($doc, $methodName)) {
                 return $doc->{$methodName}();
             }
         }
 
         // Try the magic methods __get when __isset doesn't exist
-        if (method_exists($doc, '__get') && ! method_exists($doc, '__isset')) {
+        if (method_exists($doc, '__get') && !method_exists($doc, '__isset')) {
```

Comment:
We usually keep a space after !, that's the style Revinate keeps. Are we going to change that now?

Reply:
i'd suggest to move to PSR12
that was just automatic application of IDE absed PSR12 styles.
they usually are enabled for new projects.
i beg your pardon for inconvenience 
we need to decide should we follow these PSR12 or define our own